lib/cdo/hamburger.rb

```diff
@@ -310,4 +310,117 @@ def self.get_farsi_header_contents(options)
 
     signed_out_links
   end
+
+  def self.get_farsi_hamburger_contents(options)
```

Comment:
@wilkie is planning to clean this up later but this gets it working for now with no links to the English marketing site

Reply:
Nice ok! I noticed that the `Hour of Code` link disappears on small desktop due to [this](https://github.com/code-dot-org/code-dot-org/blob/75d830bdf344ca69f3d463821e8aa3963d4fde36/pegasus/sites.v3/code.org/styles_min/040-page.css#L659-L666). If you can add a class to the farsi `#hamburger-contents` container it should be an easy fix in the css.

<img width="1050" alt="Screenshot 2024-09-20 at 9 09 16 AM" src="https://github.com/user-attachments/assets/0943f6dc-deb8-4718-b689-c1623d88bf74">